app/controllers/admin/bikes_controller.rb

```diff
@@ -142,6 +142,10 @@ def matching_bikes
     return @matching_bikes if defined?(@matching_bikes)
     bikes = Bike.unscoped
     # do example up here because it unscopes
```

Comment:
This comment is now separated from its applicable place. Which is critical in this case, because users won't work with the example query. You will need to skip example if `search_user_id`